src/app/lib/utilities/variantHandler/index.js

```diff
@@ -0,0 +1,21 @@
+const servicesWithVariants = {
+  serbian: ['lat', 'cyr'],
+  ukchina: ['simp', 'trad'],
+  zhongwen: ['simp', 'trad'],
+};
+
+// Remove leading slash from variant
+export const variantSanitzer = variant => variant && variant.replace('/', '');
+
+// If service has variants, use it or default to first variant in array
+// If service doesnt have variants, return 'default'
+export const variantDataKey = (service, rawVariant) => {
```

Comment:
The only place this function is used is in `ServiceContext/index.jsx` and there, the variant value passed in doesn't have the leading `/`. So I don't believe the sanitizer is needed here. I also suggest renaming:
```suggestion
export const getVariant = ({ service, variant }) => {
```

Reply:
Done :)